armada/cli/apply.py

```diff
@@ -39,6 +35,8 @@ def get_parser(self, prog_name):
                             help='Armada yaml file')
         parser.add_argument('--dry-run', action='store_true',
                             default=False, help='Run charts with dry run')
+        parser.add_argument('--trace', action='store_true',
```

Comment:
also for the `--debug` option, the description is given as `Run charts with dry run` is this correct?